app/controllers/recipes_controller.rb

```diff
@@ -27,6 +28,7 @@ def create
     @recipe = Recipe.new(recipe_params.merge(user_id: current_user.id))
 
     if @recipe.save
+      attach_media
```

Comment:
You should be able to update/create a recipe including it's media. IIRC there's no need to use `attach` right now

Reply:
Ok! I'm removing the explicit media attachment

Reply:
Already removed!